group_project_v2/group_project.py

```diff
@@ -11,13 +11,13 @@
 from xblock.validation import ValidationMessage
 from xblockutils.studio_editable import XBlockWithPreviewMixin, NestedXBlockSpec
 
-from group_project_v2.mixins import CommonMixinCollection
+from group_project_v2.mixins import CommonMixinCollection, DashboardXBlockMixin, DashboardRootXBlockMixin
 from group_project_v2.notifications import ActivityNotificationsMixin
 from group_project_v2.project_navigator import GroupProjectNavigatorXBlock
 from group_project_v2.utils import (
     mean, make_key, outsider_disallowed_protected_view, get_default_stage, DiscussionXBlockShim, Constants,
-    add_resource, gettext as _
-)
+    add_resource, gettext as _,
+    get_block_content_id)
```

Comment:
I think the `)` paren should be on the next line, unindented. Also, why not put `add_resource, gettext as _, get_block_content_id,` all on one line?